contracts/optimistic-ethereum/OVM/execution/OVM_ExecutionManager.sol

```diff
@@ -67,6 +67,12 @@ contract OVM_ExecutionManager is iOVM_ExecutionManager, Lib_AddressResolver {
     uint256 constant NUISANCE_GAS_PER_CONTRACT_BYTE = 100;
     uint256 constant MIN_GAS_FOR_INVALID_STATE_ACCESS = 30000;
 
+    /**************************
+     * Default Context Values *
+     **************************/
+
+    uint256 constant DEFAULT_UINT256 = 0xdefa017defa017defa017defa017defa017defa017defa017defa017defa017d;
```

Comment:
Previously in #244, I used MAX_UINT, but I feel uneasy about being 'off-by-one' from zero. 

I also wanted a number that can be reused as the default for any of the elements in a  [TransactionContext](https://github.com/ethereum-optimism/contracts/blob/d975c6e524ca9d943ff61f13e2205b0d4c083eaf/contracts/optimistic-ethereum/iOVM/execution/iOVM_ExecutionManager.sol#L47) or [MessageContext](https://github.com/ethereum-optimism/contracts/blob/d975c6e524ca9d943ff61f13e2205b0d4c083eaf/contracts/optimistic-ethereum/iOVM/execution/iOVM_ExecutionManager.sol#L60). 
I like this one, and I don't think it could ever be a valid context number.